lib/puppet_x/templates/invoke_ps_command.erb

```diff
@@ -1,15 +1,207 @@
 $hostSource = @"
 using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
 using System.Globalization;
+using System.Management.Automation;
 using System.Management.Automation.Host;
+using System.Security;
+using System.Text;
 using System.Threading;
 
-public class PuppetPSHost : PSHost
+namespace Puppet
 {
+  public class PowerShellRawUserInterface : PSHostRawUserInterface
+  {
+    public PowerShellRawUserInterface()
+    {
+      BufferSize = new Size(80, 25);
+      BackgroundColor = ConsoleColor.Black;
+      ForegroundColor = ConsoleColor.White;
```

Comment:
They were originally public auto properties that were converted to private backing variables to support .NET 2.0.